src/import/trace-event.ts

```diff
@@ -48,12 +48,30 @@ interface TraceEvent {
   cat?: string
 
   // Any arguments provided for the event. Some of the event types have required argument fields, otherwise, you can put any information you wish in here. The arguments are displayed in Trace Viewer when you view an event in the analysis section.
-  args: any
+  args?: any
```

Comment:
Based on the spec this is not a required field for some event types, so marked it as optional. Happy to change back if this was intentional

Reply:
Seems fine